app/pages/summary/contextCreator.js

```diff
@@ -316,36 +316,40 @@ const generateOneOffCostDetailsTable = ({
 
 export const getContext = ({
   orderId, orderData, oneOffCostItems, recurringCostItems, serviceRecipients,
-}) => ({
-  ...manifest,
-  title: `${manifest.title} ${orderId}`,
-  orderDescription: orderData.description,
-  dateSummaryCreated: getCurrentDate(),
-  callOffAndSupplierTable: generateCallOffAndSupplierDetailsTable({
-    callOffAndSupplierTable: manifest.callOffAndSupplierTable,
-    orderPartyData: orderData.orderParty,
-    supplierData: orderData.supplier,
-  }),
-  oneOffCostTable: generateOneOffCostDetailsTable({
-    oneOffCostTable: manifest.oneOffCostTable,
-    oneOffCostItems,
-    serviceRecipients,
-  }),
-  oneOffCostTotalsTable: generateOneOffCostTotalsTable({
-    oneOffCostTotalsTable: manifest.oneOffCostTotalsTable,
-    oneOffCostTotalValue: orderData.totalOneOffCost,
-  }),
-  recurringCostTable: generateRecurringCostDetailsTable({
-    recurringCostTable: manifest.recurringCostTable,
-    recurringCostItems,
-    serviceRecipients,
-  }),
-  recurringCostTotalsTable: generateRecurringCostTotalsTable({
-    recurringCostTotalsTable: manifest.recurringCostTotalsTable,
-    recurringYearCost: orderData.totalRecurringCostPerYear,
-    recurringMonthCost: orderData.totalRecurringCostPerMonth,
-    ownershipCost: orderData.totalOwnershipCost,
-  }),
-  commencementDate: formatDate(orderData.commencementDate),
-  backLinkHref: `${baseUrl}/organisation/${orderId}`,
-});
+}) => {
+  const manifestFromStatus = orderData.status === 'complete' ? null : manifest;
+
+  return ({
+    ...manifestFromStatus,
+    title: `${manifest.title} ${orderId}`,
+    orderDescription: orderData.description,
+    dateSummaryCreated: getCurrentDate(),
+    callOffAndSupplierTable: generateCallOffAndSupplierDetailsTable({
+      callOffAndSupplierTable: manifest.callOffAndSupplierTable,
+      orderPartyData: orderData.orderParty,
+      supplierData: orderData.supplier,
+    }),
+    oneOffCostTable: generateOneOffCostDetailsTable({
+      oneOffCostTable: manifest.oneOffCostTable,
+      oneOffCostItems,
+      serviceRecipients,
+    }),
+    oneOffCostTotalsTable: generateOneOffCostTotalsTable({
+      oneOffCostTotalsTable: manifest.oneOffCostTotalsTable,
+      oneOffCostTotalValue: orderData.totalOneOffCost,
+    }),
+    recurringCostTable: generateRecurringCostDetailsTable({
+      recurringCostTable: manifest.recurringCostTable,
+      recurringCostItems,
+      serviceRecipients,
+    }),
+    recurringCostTotalsTable: generateRecurringCostTotalsTable({
+      recurringCostTotalsTable: manifest.recurringCostTotalsTable,
+      recurringYearCost: orderData.totalRecurringCostPerYear,
+      recurringMonthCost: orderData.totalRecurringCostPerMonth,
+      ownershipCost: orderData.totalOwnershipCost,
+    }),
+    commencementDate: formatDate(orderData.commencementDate),
+    backLinkHref: `${baseUrl}/organisation/${orderId}`,
+  });
```

Comment:
the use of manifest within here would also need changing. I think it might be better to import the `'./incomplete/manifest.json';` as incompleteOrderManifest.

Then keep `manifestFromStatus` as just manifest. Then you shouldn't have to change this all here.